src/library/Uno.Material/Styles/Controls/v2/MediaPlayerElement.xaml

```diff
@@ -0,0 +1,653 @@
+														  Style="{StaticResource AppBarButtonStyle}"
+														  Foreground="{ThemeResource MediaTransportControlsAppBarButtonBrush}"
+														  AllowFocusOnInteraction="True"
+														  MediaTransportControlsHelper.DropoutOrder="19">
+												<AppBarButton.Flyout>
+													<Flyout x:Name="VolumeFlyout"
+															FlyoutPresenterStyle="{StaticResource FlyoutStyle}"
+															contract8Present:ShouldConstrainToRootBounds="False">
+														<StackPanel Orientation="Horizontal"
+																	Padding="3,11">
+															<AppBarButton x:Name="AudioMuteButton"
+																		  Style="{StaticResource MediaTransportControlsAppBarButtonStyle}"
+																		  VerticalAlignment="Center"
+																		  HorizontalAlignment="Center">
+																<AppBarButton.Resources>
+																	<Thickness x:Key="AppBarButtonContentViewboxCollapsedMargin">0,12</Thickness>
+																	<Thickness x:Key="AppBarButtonInnerBorderMargin">5</Thickness>
+																</AppBarButton.Resources>
+																<AppBarButton.Icon>
+																	<PathIcon x:Name="InnerVolumeMuteSymbol"
+																			  Data="{StaticResource VolumePathData}" />
+																</AppBarButton.Icon>
+															</AppBarButton>
+															<Slider x:Name="VolumeSlider"
+																	Value="50"
+																	IsThumbToolTipEnabled="False"
+																	Height="{ThemeResource MediaTransportControlsSliderHeight}"
+																	Width="{ThemeResource MediaTransportControlsSliderWidth}"
+																	VerticalAlignment="Center"
+																	HorizontalAlignment="Center"
+																	Margin="12,0,8,0" />
+															<TextBlock x:Name="VolumeValue"
+																	   Style="{StaticResource MediaTextBlockStyle}"
+																	   Text="{Binding ElementName=VolumeSlider, Path=Value}"
+																	   HorizontalAlignment="Center"
+																	   VerticalAlignment="Center"
+																	   TextAlignment="Right"
+																	   Width="24"
+																	   Margin="8,0,16,0" />
+														</StackPanel>
+													</Flyout>
+												</AppBarButton.Flyout>
+												<AppBarButton.Icon>
+													<PathIcon x:Name="OuterVolumeMuteSymbol"
+															  Data="{StaticResource VolumePathData}" />
+												</AppBarButton.Icon>
+											</AppBarButton>
+											<AppBarButton x:Name="CCSelectionButton"
+														  Style="{StaticResource MediaTransportControlsAppBarButtonStyle}"
+														  MediaTransportControlsHelper.DropoutOrder="15"
+														  Visibility="Collapsed">
+												<AppBarButton.Icon>
+													<FontIcon Glyph="&#xED1E;" />
+												</AppBarButton.Icon>
+											</AppBarButton>
+											<AppBarButton x:Name="AudioTracksSelectionButton"
+														  Style="{StaticResource MediaTransportControlsAppBarButtonStyle}"
+														  MediaTransportControlsHelper.DropoutOrder="13"
```

Comment:
For now let's just add `not_win:` prefixes to all of the `MediaTransportControlHelper.DropoutOrder` properties. It requires at least WinAppSDK v1.2 and we are still on 1.1.0 for Material and 1.1.5 for Toolkit

Add a comment in the file about why the not_win is there for now